### PR TITLE
add like to qr code article

### PR DESCRIPTION
**Pull request by Sterling463**
q
exit

### CODE REVIEW

**Comment by catlook** [comment]
not sure why it failed...kicking it again

**Comment by catlook** [comment]
posted in toolchain channel

File: api-docs/getting-started/use-mfa-ops.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ The following figure shows the multi-factor authentication process work flow.
 To enable multi-factor authentication, users must update the account
 settings to add and verify a device that can be used to supply the
 multi-factor authentication passcode. You can configure account settings
-from the Cloud Control panel, or by using the Identity service API.
+from the `Cloud Control panel<https://support.rackspace.com/how-to/myrackspace-multi-factor-authentication/>`_, or by using the Identity service API.
```
Comment (robb-romans) [inline api-docs/getting-started/use-mfa-ops.rst]: you need to add a space after `panel` and correct the link syntax

Comment (Sterling463) [reply api-docs/getting-started/use-mfa-ops.rst]: got it

**Comment by catlook** [comment]
build me

**Comment by Sterling463** [comment]
I don't think so @catlook . Last I heard there was an issue outside of my PR?

**Comment by robb-romans** [comment]
build me

File: api-docs/getting-started/use-mfa-ops.rst [hunk 2]
```diff
@@ -37,7 +37,8 @@ The following figure shows the multi-factor authentication process work flow.
 To enable multi-factor authentication, users must update the account
 settings to add and verify a device that can be used to supply the
 multi-factor authentication passcode. You can configure account settings
-from the Cloud Control panel, or by using the Identity service API.
+from the Cloud Control panel
+, or by using the Identity service API.
```
Comment (robb-romans) [inline api-docs/getting-started/use-mfa-ops.rst]: Please clean this up / reflow text

**Comment by catlook** [comment]
FYI: build still failing.
Traceback (most recent call last):
File "/opt/rh/rh-python36/root/usr/lib64/python3.6/runpy.py", line 193, in _run_module_as_main
"main", mod_spec)
File "/opt/rh/rh-python36/root/usr/lib64/python3.6/runpy.py", line 85, in _run_code
exec(code, run_globals)
File "/tmp/workspace/deconst-cicd-content-rackerlabs-docs-cloud-identity-pipeline/preparer-sphinx/deconstrst/main.py", line 5, in
main()
File "/tmp/workspace/deconst-cicd-content-rackerlabs-docs-cloud-identity-pipeline/preparer-sphinx/deconstrst/init.py", line 43, in main
status = build(srcdir, destdir)
File "/tmp/workspace/deconst-cicd-content-rackerlabs-docs-cloud-identity-pipeline/preparer-sphinx/deconstrst/deconstrst.py", line 32, in build
parallel=1)
File "/opt/app-root/lib/python3.6/site-packages/sphinx/application.py", line 149, in init
self.setup_extension(extension)
File "/opt/app-root/lib/python3.6/site-packages/sphinx/application.py", line 458, in setup_extension
mod = import(extension, None, None, ['setup'])
File "/opt/app-root/lib/python3.6/site-packages/chios/bolditalic/init.py", line 32, in
logger = logging.getLogger(name)
AttributeError: module 'sphinx.util.logging' has no attribute 'getLogger'
script returned exit code 1

**Comment by Sterling463** [comment]
@robb-romans it failed again. I'm not getting any errors that I recognize.

**Pull request closed without merging**